backend/src/main/java/fk/prof/backend/util/BitOperationUtil.java

```diff
@@ -0,0 +1,9 @@
+package fk.prof.backend.util;
+
+public class BitOperationUtil {
+
+  public static long constructLongFromInts(int mostSignificantBits, int leastSignificantBits) {
+    return (((long)mostSignificantBits) << 32) | (leastSignificantBits & 0xffffffffL);
```

Comment:
Actually makes more sense to keep it here. Called from tests as well to generate work id for equality comparison. Doesn't make sense for this to be exposed as a public method of aggregationwindowplanner